galleries/users_explain/axes/colorbar_placement.py

```diff
@@ -10,7 +10,11 @@
 Colorbars indicate the quantitative extent of image data.  Placing in
 a figure is non-trivial because room needs to be made for them.
 
-The simplest case is just attaching a colorbar to each axes:
+Automatic placement
+===================
```

Comment:
```suggestion
Automatic placement of colorbars
================================
```
For consistency with the "Manual" section below.  Or make the other title short to match this, but I think the longer titles would be better if other parts of the docs link to specific sections?